src/translations/en.json

```diff
@@ -1339,6 +1339,7 @@
           }
         },
         "devices": {
+          "add_prompt": " have been added using this device yet. You can add one by clicking the + button above.",
```

Comment:
We can't have half sentences in translations, because the word order in other languages might be different.

Reply:
You can use variables:
`No {name} have been added using this device yet. You can add one by clicking the + button above.`